engine/src/main/java/org/terasology/rendering/dag/nodes/BufferClearingNode.java

```diff
@@ -76,80 +79,24 @@ public void process() {
         glClear(clearingMask);
     }
 
-    /**
-     * A simple class validating and eventually providing appropriate data for
-     * the method BufferClearingNode.initialise(Object object) method.
-     */
-    public static class RequiredData {
```

Comment:
Thanks to the changes in the NodeFactory and the delayed initialization this class has been removed completely.